frontend/src/lit-components/pages/nav.js

```diff
@@ -0,0 +1,36 @@
+import { html, LitElement } from 'lit';
+
+import { NavStyles } from '../../styles';
+
+export class NavBar extends LitElement {
+    static styles = NavStyles;
+
+    static properties = {
+
+    }
+
+    constructor() {
+        super();
+    }
+
+    render() {
+        return html`
+        <header>
+            <a href="#" id="profile">
+                <img src="../../../public/images/userprofile.svg" alt="profile">
```

Comment:
src value just needs to point to `/public/images/userprofile.svg` since it's already relative to the domain.
Example, when running locally: `/public/images/userprofile.svg` will point to `http://localhost:8080/public/images/userprofile.svg`